src/layouts/lecture/styles/layout.scss

```diff
@@ -38,52 +34,66 @@
   display: none !important;
 }
 
-// Responsive grid system
 .cols,
 .columns {
   display: flex;
   flex-wrap: wrap;
   margin-right: -6px;
   margin-left: -6px;
+}
 
-  &.col-gapless {
-    margin-right: 0;
-    margin-left: 0;
-
-    & > .column {
-      padding-right: 0;
-      padding-left: 0;
-    }
-  }
+.cols.col-gapless,
+.columns.col-gapless {
+  margin-right: 0;
+  margin-left: 0;
+}
 
-  &.col-oneline {
-    flex-wrap: nowrap;
-    overflow-x: auto;
-  }
+.cols.col-oneline,
+.columns.col-oneline {
+  flex-wrap: nowrap;
+  overflow-x: auto;
 }
 
 [class~='col-'],
 .column {
   flex: 1;
   max-width: 100%;
-  padding-right: 6px;
-  padding-left: 6px;
+  padding: 8px;
+}
 
-  &.col-12,
-  &.col-11,
-  &.col-10,
-  &.col-9,
-  &.col-8,
-  &.col-7,
-  &.col-6,
-  &.col-5,
-  &.col-4,
-  &.col-3,
-  &.col-2,
-  &.col-1,
-  &.col-auto {
-    flex: none;
-  }
+[class~='col-'].col-12,
+.column.col-12,
```

Comment:
Won't this much same as 
```suggestion
.column.col-12,
```

Reply:
Yep, i mean if these selectors map same elements in our case, then we can get rid of them, but if not let's leave it as it is

Reply:
No, they are different. Frankly speaking, I would prefer to replace this layout system with something like 'Tailwind' because it fits better with the current needs of styling the 'mdx' format pages. But it will take few weeks.